cmd/bnscli/cmd_submit.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/iov-one/weave/cmd/bnsd/app"
+	"github.com/iov-one/weave/cmd/bnsd/client"
+)
+
+func cmdSubmitTx(
+	input io.Reader,
+	output io.Writer,
+	args []string,
+) error {
+	fl := flag.NewFlagSet("", flag.ExitOnError)
+	fl.Usage = func() {
+		fmt.Fprint(flag.CommandLine.Output(), `
+Read binary serialized transaction from standard input and submit it.
+
+Make sure to collect enough signatures before submitting the transaction.
+`)
+		fl.PrintDefaults()
+	}
+	var (
+		tmAddrFl = fl.String("tm", "https://bns.NETWORK.iov.one:443", "Tendermint node address. Use proper NETWORK name.")
```

Comment:
I like this, same comments as above (subdir - probably generic, and TM_ADDR environmental variable)

Reply:
Done